### ISSUE TITLE
Consistently document and tag releases

### ISSUE THREAD

**Issue by PhilippWendler** [google/code-prettify#498]
It is hard to find out what the current release of this project is:

- The [changelog](https://github.com/google/code-prettify/blob/master/CHANGES.md) has 2015-04-28 as newest date.
- The [GitHub releases page](https://github.com/google/code-prettify/releases) shows [2013-03-04](https://github.com/google/code-prettify/releases/tag/2013-03-04) as "Latest release".
- The [Git tags](https://github.com/google/code-prettify/tags) have [2015-12-04](https://github.com/google/code-prettify/releases/tag/2015-12-04) as latest release tag.
- The [documentation](https://github.com/google/code-prettify#setup) refers to [run_prettify.js](https://github.com/google/code-prettify/blob/master/loader/run_prettify.js) and [prettify-small.zip](https://github.com/google/code-prettify/blob/master/distrib/prettify-small.zip), both of which were created by c8a660b on 2017-01-13, which has the message "cut a release", but is not tagged as a release and does not mention any version number.

Please:
- update the changelog,
- assign a version number to c8a660b and create a git tag for it,
- [create "GitHub releases"](https://help.github.com/articles/creating-releases/) for all release tags, otherwise the newer tags will be hidden by default by GitHub and an older release will be shown with the green marker "Latest release".

**Comment by grokky1** [comment]
Yes very important!! I'm struggling to find out which is the latest version, and where to get it. This should be addressed so users don't end up using older versions by accident (which is what I did).

**Comment by PhilippWendler** [comment]
I see that there has been some activity on this project recently, which is nice!
Is there any chance to get this issue solved? Fixing this soon and then regularly producing new releases would greatly simplify users' lives, and for example allow to solve #507 easily.

#500 (titled "Release 10-Feb-2013") is something else which leaves me puzzled about how it relates to the above.

**Comment by PhilippWendler** [comment]
I would like to ask again whether there are any plans to do this?
Please, this would make using this library much easier.

Now at least there are sometimes updates to the ZIP in the Git repository, but (a) its some kind of anti-pattern to commit stuff like this to the development repository, (b) this is still hard to discover and confusing, and (c) a changelog would be great.

**Comment by PhilippWendler** [comment]
Note that there are sill commits every few months, so I haven't given up hope. Just unfortunate that nobody responds here.

**Comment by PhilippWendler** [comment]
I just found [this post](https://groups.google.com/forum/#!topic/js-code-prettifier/Re0QSgmbjlk) from 2016 on the project mailing list that claims it was decided that a "MAJOR.MINOR.PATCH versioning scheme" will be adopted. Please actually do this.

**Comment by rugk** [comment]
Yes, having no real release is pretty awkward…
